web/src/app/dialogs/components/DialogTitleWrapper.js

```diff
@@ -44,18 +44,18 @@ function DialogTitleWrapper(props) {
     fullScreen,
     toolbarItems,
     onClose,
-    options,
+    actions,
     subTitle,
     title,
   } = props
 
   let menu
-  if (options && options.length > 0 && fullScreen) {
-    menu = <DropDownMenu options={options} color='white' />
-  } else if (options && options.length > 0) {
+  if (actions && actions.length > 0 && fullScreen) {
+    menu = <OtherActions actions={actions} color='white' />
```

Comment:
One small thing, looks like we'll want to add the `color` functionality into `OtherActions`:

L56 `color: 'inherit',` -> `color: this.props.color ? this.props.color : 'inherit'`

and then the proptype for color